docs/user-guide/inference-backends.md

```diff
@@ -81,6 +81,17 @@ Please refer to the vLLM [documentation](https://docs.vllm.ai/en/stable/models/s
 
 vLLM supports multimodal language models listed [here](https://docs.vllm.ai/en/stable/models/supported_models.html#multimodal-language-models). When users deploy a vision language model using the vLLM backend, image inputs are supported in the chat completion API.
 
+#### Distributed Inference Across Workers
+
+vLLM supports distributed inference across multiple workers using [Ray](https://ray.io). You can enable a Ray cluster in GPUStack by using the `--enable-ray` start parameter, allowing vLLM to run distributed inference across multiple workers.
+
+!!! warning "Known Limitations"
+
+    1. Both the GPUStack server and all participating workers must run on Linux.
+    2. Model files must be accessible at the same path on all participating workers. Currently, GPUStack downloads model files only to the main worker. You must either use a shared file system or manually copy the model files to the same path on all participating workers.
+    3. Each worker can only be assigned to one distributed vLLM model instance at a time.
+    4. Only manual scheduling is supported, meaning users must select the desired workers/GPUs in the model configuration for distributed vLLM model instances.
```

Comment:
Now we also support auto scheduling with conditions.

Reply:
Updated